NetKAN/SVT-Windows.netkan

```diff
@@ -0,0 +1,26 @@
+{
+    "spec_version"   : "v1.4",
+    "$kref"          : "#/ckan/github/Galileo88/Stock-Visual-Terrain/asset_match/SVT.*(?!Mac).zip",
```

Comment:
`"#/ckan/github/Galileo88/Stock-Visual-Terrain/asset_match/SVT.*(?!Mac).zip$"`  - Without the final "$", it matches both zip file name patterns.